### PR TITLE
Add New Covered Conveyor

### PR DESCRIPTION
**Pull request by Voz-Duh**
In this pull I create 2 new types - CoveredConveyor and PayloadCoveredConveyor
This conveyors don't move units and use meta glass for make it 
covered-conveyor

### CODE REVIEW

**Comment by DeltaNedas** [comment]
stolen from routorio h

**Comment by Esnardo** [comment]
What do they do?

**Comment by Voz-Duh** [comment]
> stolen from routorio h

From Vanilla Upg >:<

**Comment by Voz-Duh** [comment]
> What do they do?

> This conveyors don't move units and use meta glass for make it

**Comment by Quezler** [comment]
this seems bloated

**Comment by DeltaNedas** [comment]
routorio payload conduit

**Comment by Anuken** [comment]
I don't think this is needed. What is the use case?

**Comment by Voz-Duh** [comment]
Screenshot

**Comment by genNAowl** [comment]
Seems to specific. Use bridges or get rid of the copper walls instead.

**Comment by Voz-Duh** [comment]
In some places you won't be able to put the bridge conveyor

**Comment by Voz-Duh** [comment]
> Get rid of the copper walls instead.

*This for example*

File: core/src/mindustry/content/Blocks.java [hunk 1]
```diff
@@ -58,8 +58,8 @@
     scrapWall, scrapWallLarge, scrapWallHuge, scrapWallGigantic, thruster, //ok, these names are getting ridiculous, but at least I don't have humongous walls yet
 
     //transport
-    conveyor, titaniumConveyor, plastaniumConveyor, armoredConveyor, distributor, junction, itemBridge, phaseConveyor, sorter, invertedSorter, router,
-    overflowGate, underflowGate, massDriver, payloadConveyor, payloadRouter,
+    conveyor, titaniumConveyor, plastaniumConveyor, coveredConveryor, armoredConveyor, distributor, junction, itemBridge, phaseConveyor, sorter, invertedSorter, router,
```
Comment (Catchears) [inline core/src/mindustry/content/Blocks.java:61]: you wrote conve**r**yor. Probably just a typo. You did the same in line 954.

**Comment by SnakkiZXZ** [comment]
I agree with Anyuk, I think that there is no point in this, since the units can cope with the usual version of pipelines. Moreover, if the conveyor is closed, then it has a volume, which means that it cannot be crossed,

**Comment by Voz-Duh** [comment]
> Moreover, if the conveyor is closed, then it has a volume, which means that it cannot be crossed,

I wanted to the units to slow down on them but then changed my mind

**Comment by Voz-Duh** [comment]
Ok

**Comment by Voz-Duh** [comment]
I can close it

**Pull request closed without merging**